Municipios_Brasileiros.csv

```diff
@@ -5549,17 +5549,23 @@ Código IBGE;Nome do Município;Código UF;UF;Estado;Latitude;Longitude
 5106158; Nova Bandeirantes; 51; MT; Mato Grosso; -9.8497658; -57.8138733
 5107800; Santo Antônio do Leverger; 51; MT; Mato Grosso; -15.8631830; -56.0788116
 5105622; Mirassol d'Oeste; 51; MT; Mato Grosso; -15.6759472; -58.0951385
-5105234; Lambari d'Oeste; 51; MT; Mato Grosso; -15.3188457; -58.0046234
-5107008; Poxoréo; 51; MT; Mato Grosso; -15.8299230; -54.4208067
+5105234; Lambari D'Oeste; 51; MT; Mato Grosso; -15.3188457; -58.0046234
+5107008; Poxoréu; 51; MT; Mato Grosso; -15.8299230; -54.4208067
 5104526; Ipiranga do Norte; 51; MT; Mato Grosso; -12.2407999; -56.1531029
-5103957; Glória d'Oeste; 51; MT; Mato Grosso; -15.7680092; -58.3107605
+5103957; Glória D'Oeste; 51; MT; Mato Grosso; -15.7680092; -58.3107605
 5104542; Itanhangá; 51; MT; Mato Grosso; -12.2259455; -56.6462517
-5103809; Figueirópolis d'Oeste; 51; MT; Mato Grosso; -15.4439182; -58.7391472
+5103809; Figueirópolis D'Oeste; 51; MT; Mato Grosso; -15.4439182; -58.7391472
 5105507; Vila Bela da Santíssima Trindade; 51; MT; Mato Grosso; -15.0067739; -59.9504280
-5103361; Conquista d'Oeste; 51; MT; Mato Grosso; -14.5381174; -59.5443764
+5103361; Conquista D'Oeste; 51; MT; Mato Grosso; -14.5381174; -59.5443764
 5003900; Figueirão; 50; MS; Mato Grosso do Sul; -18.6782093; -53.6379852
 4212809; Balneário Piçarras; 42; SC; Santa Catarina; -26.7638893; -48.6716652
 4216909; São Lourenço do Oeste; 42; SC; Santa Catarina; -26.3556595; -52.8498383
 4207684; Ipuaçu; 42; SC; Santa Catarina; -26.6349869; -52.4555931
 4213906; Presidente Castello Branco; 42; SC; Santa Catarina; -27.2217865; -51.8089485
-2613107; São Caetano; 26; PE; Pernambuco; -8.3376271; -36.2868983
+2613107; São Caitano; 26; PE; Pernambuco; -8.3376271; -36.2868983
```

Comment:
I guess I was going off of [this](http://cidades.ibge.gov.br/xtras/perfil.php?codmun=261310) for reference?